### ISSUE TITLE
Some bound events for inputmaps don't do anything

### ISSUE THREAD

**Issue by Rainbowlemon** [jellyfin/jellyfin-media-player#556]
**Describe the bug**
I've been setting up a new set of inputmaps for my xbox 360 wireless controller, and realised that a few events just don't have an action associated with the command. (I've copied some events from the other inputmaps examples). E.g:

```
2024-02-02 12:30:48.668 [debug] unknown @ 0 - Input received: source: "360 Wireless Receiver (XBOX)" keycode: "KEY_BUTTON_8" : InputBase::KeyDown
2024-02-02 12:30:48.669 [debug] unknown @ 0 - No match for: "360 Wireless Receiver (XBOX)"
2024-02-02 12:30:48.669 [debug] unknown @ 0 - No match for: "KEY_BUTTON_8"
2024-02-02 12:30:50.279 [debug] unknown @ 0 - Input received: source: "360 Wireless Receiver (XBOX)" keycode: "KEY_BUTTON_8" : InputBase::KeyUp
2024-02-02 12:30:50.280 [debug] unknown @ 0 - "Emit input action (long):" "exit"
2024-02-02 12:30:50.282 [info] unknown @ 0 - JS: inputManager: tried to process command with no action assigned: exit
```

This happens for `exit`, `increase_audio_delay`, `decrease_audio_delay` and `toggle_subtitles` - none of them seem to trigger. Oddly, it's throwing the 'No match' error out even though I do get a match for inputs on my controller.

Not sure if the input binding issues are related, but I also have issues with input repeats not working

**To Reproduce**
1. Ensure a controller works with Jellyfin Media Player and bind a key to one of the above events. `increase_audio_delay` is already bound to right on the right stick for a few controllers
2. Go to any media and press play
3. Try and use the bound event

**Expected behavior**
Bound event should work

**Desktop (please complete the following information):**
 - OS: Zorin 17 (Ubuntu 22.04 base)
 - Installation Method: Flatpak, connecting to a jellyfin server running on docker on the same host

**Additional info**
I'm also getting odd duplicate input events that don't have a match (see in example above), though I don't think this is related.

**Comment by sevenrats** [comment]
*how* exactly have you configured these input maps? are you using the client in TV mode?

**Comment by Rainbowlemon** [comment]
Yeah, client is in TV mode. I've copied the xbox controller json file from the inputmaps examples to the parent directory, renamed it, and got it to match my controller & rebound keys. The issue isn't with the controller not working - it works for the most functions, like back/forward, play, etc. Just these other functions don't seem to do anything.

For example, I'd like to rebind the xbox home button to exit the media player, but the `exit` action doesn't seem to do anything.  Neither does `host:exit`.

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
